Source/WebCore/page/Quirks.cpp

```diff
@@ -166,15 +166,23 @@ bool Quirks::needsFormControlToBeMouseFocusable() const
     if (!needsQuirks())
         return false;
 
-    auto host = topDocumentURL().host();
-    if (host == "ceac.state.gov"_s || host.endsWith(".ceac.state.gov"_s))
-        return true;
+    if (!m_quirksData.needsFormControlToBeMouseFocusableQuirk) {
+        m_quirksData.needsFormControlToBeMouseFocusableQuirk = [&] {
+            auto host = topDocumentURL().host();
+            if (host == "ceac.state.gov"_s || host.endsWith(".ceac.state.gov"_s))
+                return true;
 
-    if (host == "weather.com"_s)
-        return true;
-#endif // PLATFORM(MAC)
+            if (host == "weather.com"_s)
+                return true;
 
+            return false;
+        }();
+    }
+
+    return *m_quirksData.hasBrokenEncryptedMediaAPISupportQuirk;
```

Comment:
I might be just misreading the diff tool, but this looks like the wrong cached bit?

```suggestion
    return *m_quirksData.needsFormControlToBeMouseFocusableQuirk;
```

Reply:
Oh, gosh! Yes -- I messed this up. Will fix. Thank you for catching that!